book/src/dev/rfcs/0005-state-updates.md

```diff
@@ -199,19 +199,19 @@ This means that our external API ensures that the state service sees a
 linearized sequence of state requests, although the exact ordering is
 unpredictable when there are multiple senders making requests.
 
-In the second category, the Sled API presents itself synchronously, but
+In the second category, the rocksdb API presents itself synchronously, but
 database and tree handles are cloneable and can be moved between threads. All
 that's required to process some request asynchronously is to clone the
 appropriate handle, move it into an async block, and make the call as part of
 the future. (We might want to use Tokio's blocking API for this, but this is
 an implementation detail).
```

Comment:
I think this paragraph might not be accurate any more after replacing "sled" with "rocksdb"?

Reply:
I don't think it's essential -- I think we could just delete this?